examples/transformers/README.md

```diff
@@ -0,0 +1,189 @@
+| bert-base-uncased 	|0        	|1                  	|no       |no        |0            	|09.685     |3.614      |
+| bert-base-uncased 	|0        	|2                  	|no       |no        |0            	|88.002     |80.634     |
+| bert-base-uncased 	|0        	|10                 	|no       |no        |0            	|87.603     |79.130     |
+| bert-base-uncased 	|80       	|1                  	|yes      |yes       |0          	|25.141     |15.998     |
+| bert-base-uncased 	|80       	|2                   	|yes      |no        |0            	|06.068    	|00.312     |
+| bert-base-uncased 	|80       	|10                  	|yes      |no        |8          	|83.951     |74.409     |
+| bert-base-uncased 	|90       	|1                  	|yes      |yes       |0           	|16.064     |07.786     |
+| bert-base-uncased 	|90       	|2                   	|yes      |no        |0            	|64.185     |50.946     |
+| bert-base-uncased 	|90       	|10                 	|yes      |no        |8            	|79.091     |68.184     |
+| bert-base-uncased 	|95       	|1                  	|yes      |yes       |0           	|10.501     |4.929      |
+| bert-base-uncased 	|95       	|2                   	|yes      |no        |0            	|24.445     |14.437     |
+| bert-base-uncased 	|95       	|10                 	|yes      |no        |8            	|72.761  	|60.407     |
+| bert-base-uncased 	|99         |1                   	|yes      |yes       |0             |09.685     |03.614     |
+| bert-base-uncased 	|99       	|2                   	|yes      |no        |0            	|17.433     |07.871     |
+| bert-base-uncased 	|99         |10                    	|yes      |no        |8             |47.306    	|32.564     |
+
+
+## Script origin and how to integrate sparseml with other Transformers projects
+This script is based on the example BERT-QA implementation in transformers found [here](https://github.com/huggingface/transformers/blob/master/examples/question-answering/run_qa.py). 
+
+For any other projects combining huggingface transformer's there are essentially four components to modify: imports and needed function, loading sparseml, modifying training script, and onnx export. 
+
+First take your existing project and add the following imports and functions
+```python
+from transformers.optimization import (
+    Adafactor,
+    AdamW,
+    get_constant_schedule,
+    get_constant_schedule_with_warmup,
+    get_cosine_schedule_with_warmup,
+    get_cosine_with_hard_restarts_schedule_with_warmup,
+    get_linear_schedule_with_warmup,
+    get_polynomial_decay_schedule_with_warmup,
+)
+
+from sparseml.pytorch.optim.manager import ScheduledModifierManager
+from sparseml.pytorch.optim.optimizer import ScheduledOptimizer
+from sparseml.pytorch.utils import ModuleExporter
+from sparseml.pytorch.helpers import any_str_or_regex_matches_param_name
+
+
+def get_sparsity_by_regex(module: Module, param_names: List[str]):
```

Comment:
We have functionality similar to this already in sparseML, let's move this function into sparseML